CONTRIBUTING.md

```diff
@@ -0,0 +1,194 @@
+
+#### Writing conventions
+
+##### Be lazy
+
+###### No braces `{}`:
+```stylus
+body
+    font-size: 16px;
+```
+
+###### No semi-colons `;`:
+```stylus
+body
+    font-size: 16px
+```
+
+###### And no colons either `:`:
+```stylus
+body
+    font-size 16px
+```
+
+##### Indentation
+Use 4 spaces indentation
+
+##### Nesting
+Use nested declarations for better readability:   
+```stylus
+.component
+    /* .component styles here */
+
+    &-list
+      /* .component-list styles here */
+
+      > li
+        /* .component styles here */
+```
+Instead of:
+```stylus
+.component
+    /* .component styles here */
+
+.component-list
+    /* component-list styl here */
+
+.component-list > li
+    /* .component-list > li styl here */
+```
+
+##### Variables
+Make them start with a `$` sign, e.g.:
+```stylus
+$color-base = #4c4d4e  // Default text color
+```
+
+##### Classes naming
+- Use english names in camelCase if composed
+- Use hyphens (`-`) to separe component name to modifier or child name
+- Avoid content or context related classe name, it should be related to UI
+
+#### Pseudo classes/elements
+- `:after` and `:before` -> Use single colon notation to ensure IE8 support
+- `:last-child` is not supported by IE8 -> Deal with `:first-child` instead if possible
+
+#### Colors
+- Must be a variable
+- Use hexadecimal format
+- Lowercase
+- Short notation if possible
+
+Write `#fff` instead of `#ffffff` or `#FFFFFF` or `#FFF`.
+
+#### Mixins first
+Declare mixins before CSS properties:
+```stylus
+.foo
+    _myMixin()
+    display block
+```
+Instead of:
+```stylus
+.foo
+    display block
+    _myMixin()
+```
+
+#### Use standard CSS
+If you need some future css, prefixed or add an hack, let this job to PostCSS.
+
+#### Stylus files organisation
+
+- __Settings__: Variables of the projects
+- __Mixins__: Stylus mixins
+- __@blocks__: Stylus blocks for factorisation related to design system
+- __Elements__: Indivisible UI element (named “atom” in Atomic Design method)
+- __Components__: Bloc of UI, it can contain or be composed by other elements or components (named “molecule” in Atomic Design method.
+- __Themes__: Classes able to alter colors of some components and/or elements
+- __Helpers__: Small utils classes with only one css property
+
+## Code of Conduct
+
+This Code of Conduct is adapted from the [Contributor Covenant](http://contributor-covenant.org), [version 1.4](http://contributor-covenant.org/version/1/4)
+
+### Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
+orientation.
+
+### Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
```

Comment:
includes:

Reply:
It’s not from me, but from the source http://contributor-covenant.org/version/1/4 are you sure ?

Reply:
My bad ! Exemple**s** (plural)